app/services/piv_cac_service.rb

```diff
@@ -70,7 +70,9 @@ def token_response(token)
         ssl_config = { verify: !FeatureManagement.identity_pki_local_dev? }
       end
 
-      Faraday.new(ssl: ssl_config).post(
+      Faraday.new(ssl: ssl_config) do |f|
+        f.request :instrumentation
```

Comment:
should we add this to all the `Faraday.new` callsites? Or maybe add some shared Faraday helper so we can centralize/add it once?

```
identity-idp:main> git grep -l Faraday.new -- app 
app/jobs/job_helpers/faraday_helper.rb
app/services/outbound_health_checker.rb
app/services/piv_cac_service.rb
app/services/push_notification/http_push.rb
app/services/usps_in_person_proofer.rb
```

Reply:
👍🏼 

I hadn't realized there were more, so I'm glad you caught them. The decision point for me was whether we'd ever want an alarm or would need them in a dashboard-friendly way.